README.md

```diff
@@ -38,6 +38,7 @@ policies:
     dco: true
     gpg: true
     imperative: true
+    anyScope: false
```

Comment:
Discussion point: Is there a better name for configuration setting?

Reply:
Maybe `allowAnyScope`?